libraries/sql/_connection.rb

```diff
@@ -90,7 +90,11 @@ def install_pg_gem
               declare_resource(:package, libpq_package_name) { compile_time(true) }
               declare_resource(:package, 'perl-IPC-Run') do
                 compile_time(true)
-                options('--enablerepo=crb')
+                if platform?('oracle')
```

Comment:
Should we be setting this only for Oracle 9 rather than all versions?

Reply:
good catch! Updated.